root/artist/CannotSplit.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l} from '../static/scripts/common/i18n';
+
+import ArtistLayout from './ArtistLayout';
+
+const CannotSplit = ({artist}: {artist: ArtistT}) => (
+  <ArtistLayout entity={artist} page="cannot_split">
```

Comment:
ArtistLayout requires page. We could just make that optional instead, if preferred (tt didn't pass a page here)

Reply:
In fact, it goes all the way up to EntityHeader requiring page. Could still be changed, but there might be a specific reason this was done that way :)

Reply:
Looks like it could technically be made optional, though it doesn't hurt much to specify it either.